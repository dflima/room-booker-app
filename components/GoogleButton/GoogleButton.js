import React from "react";
import PropTypes from "prop-types";
import { Button, StyleSheet } from "react-native";

const GoogleButton = ({ title, onPress }) => (
  <Button style={styles.button} title={title} onPress={onPress} />
);

GoogleButton.propTypes = {
  title: PropTypes.string.isRequired,
  onPress: PropTypes.func
};

const styles = StyleSheet.create({
  button: {
    width: 184,
    height: 42,
    backgroundColor: "#4285f4",
    fontSize: 48
  }
});

export default GoogleButton;
