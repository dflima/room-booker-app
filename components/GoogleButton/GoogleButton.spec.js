import React from "react";
import renderer from "react-test-renderer";
import GoogleButton from "./GoogleButton";

describe("GoogleButton", () => {
  it("should match the snapshot", () => {
    const { container } = renderer
      .create(<GoogleButton title="Title" />)
      .toJSON();
    expect(container).toMatchSnapshot();
  });
});
