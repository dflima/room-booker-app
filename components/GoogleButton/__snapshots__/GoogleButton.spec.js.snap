// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoogleButton should match the snapshot 1`] = `undefined`;
